com.trollworks.gcs/src/com/trollworks/gcs/menu/item/OpenPageReferenceCommand.java

```diff
@@ -117,10 +117,25 @@ public static void openReference(String reference, String highlight) {
                 }
             }
             if (ref != null) {
+                //If the user has specified a custom PDF viewer string, use that
```

Comment:
I definitely tend err on the side of more comments. I usually use them as a sort of header for a "paragraph" of code, so that you can navigate the code without having to read every line - I like the "intent in comments, implementation in code". 

Will remove.

Reply:
I'm not against comments... I just prefer them to say more than the code does. In addition to this, I very much dislike comments that get out of sync with the code (i.e. essentially saying the code does something it does not, or performs in a different way than it actually does), which tends to be incredibly common, especially when the comment really doesn't impart any additional information.

Reply:
I can understand that, but my experience is that it's usually easier to write code than read it, and comments that indicate intent without implementation are less likely to become out of date.